manifests/server/config.pp

```diff
@@ -35,6 +35,13 @@
     mode   => '0700',
   }
 
+  file { $mcollective::ssldir:
```

Comment:
Might want to throw is inside of an if block, so people can choose to manage it or not :+1: